README.md

```diff
@@ -17,22 +17,14 @@ This application is intended to introduce the main features of Cesium in context
 Local server
 ------------
 
-A local HTTP server is required to run the app.
+This app comes with a simple server ([`server.js`](./server.js)), but can be served through any means.
 
-Have python installed?  If so, from the `cesium-workshop` root directory run
```

Comment:
I think we did this because GIS developers are more likely to have python installed locally that node.  I would leave it as is

Reply:
The workshop specified the Node workflow. I'll provide python as an alternative, but I think the Node one should go first to not confuse people referred from the the workshop.

Reply:
Sounds good